### PR TITLE
Styling fixes and layout improvements

### PR DESCRIPTION
**Pull request by TodePond**
This PR does a bunch of styling fixes and layout tweaks. 
I removed a few buttons too - I've been trying to simplify so that we can squeeze more in.

image

### CODE REVIEW

File: app/components/JoinForm.tsx [hunk 1]
```diff
@@ -128,7 +128,7 @@ const JoinForm = () => {
 			</div>
 
 			<div className="input-graphic">
-				<Image alt="" src="/images/pollsAI.png" width={641} height={507} />
```
Comment (KaustubhKumar05) [inline app/components/JoinForm.tsx:131]: Any particular reason for this change @TodePond ?

Comment (KaustubhKumar05) [reply app/components/JoinForm.tsx:131]: Got it, makes sense

**Comment by TodePond** [comment]
Thanks for the reviews! @adityathakurxd I restored the Regenerate button!

**Review by adityathakurxd — APPROVED** [review]
Thank you for the changes 🙌

**Pull request merged**